lib/cybersource/ext/active_merchant/active_merchant.rb

```diff
@@ -35,6 +35,8 @@ def add_auth_service(xml, payment_method, options)
             xml.tag!("commerceIndicator", options[:commerce_indicator]) unless options[:commerce_indicator].blank?
           end
         end
+        # Add support for Android Pay
+        xml.tag!("paymentSolution", @options[:payment_solution]) unless options[:payment_solution].blank?
```

Comment:
Something seems off: `@options` vs `options`?

In general though, I don't think this is the right approach. `paymentSolution` is an implementation detail of the CyberSource API, the caller shouldn't have to know about it. Could you use the [source](https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/network_tokenization_credit_card.rb#L22) method instead to populate this value?